soc/v2/st_stm32/common/Kconfig.defconfig.series

```diff
@@ -0,0 +1,60 @@
+# ST Microelectronics STM32 all MCU lines
```

Comment:
have been renaming Kconfig.defconfig.series to Kconfig.defconfig in my converts @tejlmand

Reply:
ok, done